guest/abi/internal/imports/imports.go

```diff
@@ -0,0 +1,15 @@
+//go:build tinygo.wasm
+
+package imports
+
+//go:wasm-module k8s.io/scheduler
+//go:export reason
+func _reason(ptr, size uint32)
+
+//go:wasm-module k8s.io/api
+//go:export nodeInfo/node/name
+func _nodeInfoNodeName(ptr uint32, limit bufLimit) (len uint32)
```

Comment:
In both implementations, I am choosing nodeInfo/node/name and pod/spec just to make the demo work. I don't expect this to be the final form of the plugin. The main thing is to defer unmarshalling until requested. So a complete impl may just have 2 functions `nodeInfo` and `pod`. The nice thing about having a base case is being able to compare before/after decisions (because benchmark are already here).